homeassistant/components/python_script/strings.json

```diff
@@ -4,5 +4,11 @@
       "name": "[%key:common::action::reload%]",
       "description": "Reloads all available Python scripts."
     }
+  },
+  "issues": {
+    "script_continue_on_error": {
+      "title": "'python_script.{script_name}' logs errors and continues",
+      "description": "The `python_script` integration will change to raise script errors instead of logging them.\n\nThis will **stop** automations/scripts if `continue_on_error` is not set ([docs](https://www.home-assistant.io/docs/scripts/#continuing-on-error)).\n\nTo enable the new behavior, add a `response_variable` ([docs](https://www.home-assistant.io/docs/scripts/service-calls/#use-templates-to-handle-response-data)) to your service call."
```

Comment:
We only create repair issues if there's a clear action for the user to take to resolve the issue. It's not clear to me that the user always has a clear action to resolve the issue. The script may have failed temporarily and the user may not want to continue on error. A response variable isn't always usable.

Reply:
True, didn't think of this. Thanks!